app/assets/stylesheets/moderators.scss

```diff
@@ -213,19 +213,38 @@
     margin: 90px auto;
     border-radius: 12px;
   }
+
+  // Reaction Buttons START
+
+  .thumb-reactions {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin-top: 20px;
```

Comment:
Let's use the space unit variables here. :) Lisa's comments on my other PR were really helpful here: https://github.com/thepracticaldev/dev.to/pull/7395

With space unit variables, this would be `margin-top: $su-5;`. You can find them in `_import.scss`